include/aspect/material_model/visco_plastic.h

```diff
@@ -114,7 +114,9 @@ namespace aspect
         void
         parse_parameters (ParameterHandler &prm);
 
-      private:
+        double get_min_strain_rate() const;
```

Comment:
@gassmoeller - It is currently not used by the particle properties or postprocessors, but it may well be in the future so I've made it public.